src/invertColor.js

```diff
@@ -13,7 +13,21 @@
  * @return {string}
  */
 function invertColor(color) {
-  // write code here
+  let zero = '';
+
+  const hexMax = 0xffffff;
+  const hexActive = '0x' + color.slice(1);
+  const hexInvert = (hexMax - hexActive).toString(16);
```

Comment:
You can't use `toString` :)  I used it and my solution wasn't approved. It's considered cheating